CHANGES.md

```diff
@@ -1389,6 +1389,14 @@ breaking changes, and mappings for the large list of deprecated functions.
 
    *Raja Ashok*
 
+ * Added a new concept for OpenSSL plugability: providers.  This
+   functionality is designed to replace ENGINE, and to be much more
```

Comment:
maybe `ENGINEs` or `ENGINE API`?

Reply:
Also I am unsure about the comma after `ENGINE`. It seems to confuse the sentence.

Reply:
I went with "ENGINE API and ENGINE implementations"